main.nf

```diff
@@ -18,19 +18,42 @@ def helpMessage() {
 
     The typical command for running the pipeline is as follows:
 
-    nextflow run nf-core/viralrecon --reads '*_R{1,2}.fastq.gz' -profile docker
+    nextflow run nf-core/viralrecon --input samplesheet.csv --genome hg38 -profile docker
 
     Mandatory arguments:
-      --reads [file]                Path to input data (must be surrounded with quotes)
-      -profile [str]                Configuration profile to use. Can use multiple (comma separated)
-                                    Available: conda, docker, singularity, test, awsbatch, <institute> and more
-
-    Options:
-      --genome [str]                  Name of iGenomes reference
-      --single_end [bool]             Specifies that the input is single-end reads
+      --input [file]                  Comma-separated file containing information about the samples in the experiment (see docs/usage.md)
+      --protocol [str]                Specifies the type of protocol used for sequencing i.e. "metagenomics" or "amplicon". (Default: "metagenomics")
+      --host_fasta [file]             Path to Fasta reference for host genome. Not mandatory when using reference in iGenomes config via --genome
+      --viral_fasta [file]            Path to Fasta reference for viral genome
+      -profile [str]                  Configuration profile to use. Can use multiple (comma separated)
+                                      Available: conda, docker, singularity, test, awsbatch, <institute> and more
 
     References                        If not specified in the configuration file or you wish to overwrite any of the references
-      --fasta [file]                  Path to fasta reference
+      --genome [str]                  Name of iGenomes reference for host genome
+      --host_bowtie2_index [file]     Full path to directory containing Bowtie2 index including base name for host genome i.e. /path/to/index/genome.fa
```

Comment:
I'm guessing it's the same for this one.
So `bowtie2` instead of `bowtie2_index`, and also I would make that more explicit that `--host_bowtie2` can actually be just `--bowtie2`

Reply:
Maybe make a section for host reference and viral reference.

Have we thought of `--host_genome` and `--viral_genome`?

Isn't it nicer to have `--genome_host` and `genome_viral`? (same for all params actually)

Reply:
Yeah, the more I think about it the more I am realising that having `host_*` and `viral_*` parameters for everything makes sense. It will just be more explicit and easier to understand for end users.

Reply:
Fixed for now https://github.com/nf-core/viralrecon/pull/11/commits/741e9e60367b05b11d2fc448ef67e9687129ae00
Lets see how the pipeline takes shape and then we can rename the parameters as required 👍 

I did think about the `genome_host` and `genome_viral` syntax but its much easier to group them visually if using `host_genome`, `host_fasta` etc.